docs/_docs/FAQ.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Frequently Asked Questions
+category: FAQ
+chapter: 10
+order:
+---
+
+Frequently asked questions about Dependency Track functionality that may not be covered by the documentation. If you don't find an answer here, try reaching out to the Slack [channel](https://owasp.slack.com/archives/C6R3R32H4) related to dependency track.
+
+#### Dependency Check and Dependency Track Comparison
+
+This topic is heavily explained in the [Dependency Check Comparison](./../odt-odc-comparison/) to Dependency Track.
+
+#### I expect to see vulnerable components but I don't
+
+Most common reason: You have to enabled the [Sonatype OSS Index Analyzer](./../datasources/ossindex/). It is not
```

Comment:
Suggest: "You have not yet enabled" or "You have yet to enable"

Reply:
Went with the second version as it forms a positive sentence. Thanks